sources/logdog/src/log_request.rs

```diff
@@ -91,19 +94,31 @@ where
     P: AsRef<Path>,
 {
     let request = request.as_ref();
-    // get the first and second token: i.e. mode and output filename, put the remainder of the
-    // log request into the instructions field (or default to an empty string).
-    let mut iter = request.splitn(3, ' ');
-    let req = LogRequest {
-        mode: iter.next().context(error::ModeMissing)?,
-        filename: iter.next().context(error::FilenameMissing { request })?,
-        instructions: iter.next().unwrap_or(""),
+    let mut conf = request.splitn(2, ' ');
+    let mode = conf.next().context(error::ModeMissing)?;
+    let req = if mode == "glob" {
+        // for glob request format is "glob <pattern>"
+        LogRequest {
+            mode,
+            filename: "",
+            instructions: conf.next().context(error::PatternMissing)?,
+        }
+    } else {
+        // Get the second token (output filename) and put the remainder of the
+        // log request into the instructions field (or default to an empty string).
+        let mut iter = request.splitn(3, ' ');
```

Comment:
```suggestion
        let mut iter = request.splitn(2, ' ');
```
Already split out the mode.

Reply:
> Would you please confirm you've re-tested what's in the PR description?

I did test it before the last push, there was only minor change after that, sorry that was a miss. I will re-test.

Reply:
re-tested everything, CI is also passing now.